react/src/components/ResourceAllocationFormItems.tsx

```diff
@@ -71,6 +71,7 @@ type MergedResourceAllocationFormValue = ResourceAllocationFormValue &
 interface ResourceAllocationFormItemsProps {
   enableNumOfSessions?: boolean;
   enableResourcePresets?: boolean;
+  showWarningMsg?: boolean;
```

Comment:
How about renaming to `showRemainingWarning`?

Reply:
Good Idea.